src/session.ts

```diff
@@ -23,11 +23,23 @@
 import {ServiceObject} from '@google-cloud/common-grpc';
 import {promisifyAll} from '@google-cloud/promisify';
 import * as extend from 'extend';
-import * as is from 'is';
 import * as r from 'request';
 import {Transaction} from './transaction';
 import {Database} from './database';
-import {ServiceObjectConfig, DeleteCallback, Metadata, GetMetadataCallback} from '@google-cloud/common';
+import {ServiceObjectConfig, DeleteCallback, Metadata, GetMetadataCallback, ResponseCallback} from '@google-cloud/common';
+
+export type GetSessionResponse = [Session, r.Response];
+
+export interface GetSessionCallback {
+  (err: Error|null, session: Session|null, apiResponse: r.Response): void;
```

Comment:
Both `session` and `apiResponse` can be optional here if an error is thrown.  Lets make this:

```js
 (err: Error|null, session?: Session|null, apiResponse?: r.Response): void
```

Follow this pattern here and below!

Reply:
Change done.